### PR TITLE
php unit tests 3 sprint

### PR DESCRIPTION
**Pull request by wiewiurdp**

### CODE REVIEW

File: module/event-sourcing/tests/Domain/Entity/MultimediaTest.php [hunk 1]
```diff
@@ -0,0 +1,93 @@
+<?php
+
+/**
+ * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
+ * See LICENSE.txt for license details.
+ */
+
+declare(strict_types = 1);
+
+namespace Ergonode\EventSourcing\Tests\Domain\Entity;
```
Comment (rprzedzik) [inline module/event-sourcing/tests/Domain/Entity/MultimediaTest.php]: There is something wrong with namespace

File: module/event-sourcing/tests/Domain/Entity/ProcessorTest.php [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    {
+        $processor = new Processor($this->id, $this->transformerId, $this->importId, $this->action);
+        $processor->process();
+        $processor->process();
+    }
+
+    /**
+     * @expectedException \LogicException
+     */
+    public function testEnd()
```
Comment (rprzedzik) [inline module/event-sourcing/tests/Domain/Entity/ProcessorTest.php]: Missing return type

File: module/event-sourcing/tests/Domain/Entity/ProcessorTest.php [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+    {
+        $processor = new Processor($this->id, $this->transformerId, $this->importId, $this->action);
+        $processor->process();
+        $processor->end();
+        $processor->end();
+    }
+    /**
+     * @expectedException \LogicException
+     */
+    public function testStop()
```
Comment (rprzedzik) [inline module/event-sourcing/tests/Domain/Entity/ProcessorTest.php]: Missing return type

File: module/event-sourcing/tests/Domain/Entity/TransitionTest.php [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+    private $conditionSetId;
+
+    /**
+     * @var RoleId[]
+     */
+    private $roleIds;
+
+    /**
+     */
+    protected function setUp()
```
Comment (rprzedzik) [inline module/event-sourcing/tests/Domain/Entity/TransitionTest.php]: Missing return type

File: module/event-sourcing/tests/Infrastructure/Handler/ChangeProductAttributeValueCommandHandlerTest.php [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+     * @var ValueManipulationService
+     */
+    private $service;
+
+    /**
+     * @var AttributeRepositoryInterface
+     */
+    private $attributeRepository;
+
+    private $language;
```
Comment (rprzedzik) [inline module/event-sourcing/tests/Infrastructure/Handler/ChangeProductAttributeValueCommandHandlerTest.php]: Missing phpDoc

File: module/event-sourcing/tests/Infrastructure/Handler/ChangeProductAttributeValueCommandHandlerTest.php [hunk 6]
```diff
@@ -0,0 +1,137 @@
...
+
+    private $command;
```
Comment (rprzedzik) [inline module/event-sourcing/tests/Infrastructure/Handler/ChangeProductAttributeValueCommandHandlerTest.php]: Missing phpDoc

File: module/event-sourcing/tests/Infrastructure/Handler/ChangeProductAttributeValueCommandHandlerTest.php [hunk 7]
```diff
@@ -0,0 +1,137 @@
...
+        $this->language = $this->createMock(Language::class);
+        $this->command = $this->createMock(ChangeProductAttributeValueCommand::class);
+        $this->command->method('getLanguage')->willReturn($this->language);
+        $this->command->method('getId')->willReturn($this->createMock(ProductDraftId::class));
+        $this->command->method('getAttributeId')->willReturn($this->createMock(AttributeId::class));
+    }
+
+    /**
+     */
+    public function testInvokeRemovingAttr()
```
Comment (rprzedzik) [inline module/event-sourcing/tests/Infrastructure/Handler/ChangeProductAttributeValueCommandHandlerTest.php]: Missing return type

File: module/event-sourcing/tests/Infrastructure/Handler/ChangeProductAttributeValueCommandHandlerTest.php [hunk 8]
```diff
@@ -0,0 +1,137 @@
...
+     */
+    private $attributeRepository;
+
+    private $language;
+
+    private $command;
+
+    /**
+     */
+    public function setUp()
```
Comment (rprzedzik) [inline module/event-sourcing/tests/Infrastructure/Handler/ChangeProductAttributeValueCommandHandlerTest.php]: Missing return type

File: module/event-sourcing/tests/Infrastructure/Handler/ChangeProductAttributeValueCommandHandlerTest.php [hunk 9]
```diff
@@ -0,0 +1,137 @@
...
+    }
+
+    /**
+     * @param mixed  $value
+     * @param string $type
+     * @param bool   $multilingual
+     *
+     * @dataProvider dataProvider
+     */
+    public function testInvokeNewValue($value, string $type, bool $multilingual)
```
Comment (rprzedzik) [inline module/event-sourcing/tests/Infrastructure/Handler/ChangeProductAttributeValueCommandHandlerTest.php]: Missing return type

File: phpunit.xml.dist [hunk 10]
```diff
@@ -32,7 +32,6 @@
             <directory>./module/*/src/Application</directory>
             <directory>./module/*/src/Domain</directory>
             <directory>./module/*/src/Infrastructure</directory>
-            <directory>./module/grid/src/*</directory>
```
Comment (rprzedzik) [inline phpunit.xml.dist]: Why you remove it ???

File: module/event-sourcing/tests/Domain/Entity/ProcessorTest.php [hunk 11]
```diff
@@ -0,0 +1,95 @@
+<?php
+
+/**
+ * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
+ * See LICENSE.txt for license details.
+ */
+
+declare(strict_types = 1);
+
+namespace Ergonode\EventSourcing\Tests\Domain\Entity;
```
Comment (rprzedzik) [inline module/event-sourcing/tests/Domain/Entity/ProcessorTest.php]: There is something wrong with namespace

File: module/event-sourcing/tests/Domain/Entity/TransitionTest.php [hunk 12]
```diff
@@ -0,0 +1,118 @@
+<?php
+
+/**
+ * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
+ * See LICENSE.txt for license details.
+ */
+
+declare(strict_types = 1);
+
+namespace Ergonode\EventSourcing\Tests\Domain\Entity;
```
Comment (rprzedzik) [inline module/event-sourcing/tests/Domain/Entity/TransitionTest.php]: There is something wrong with namespace

File: module/event-sourcing/tests/Domain/Entity/TransitionTest.php [hunk 13]
```diff
@@ -0,0 +1,118 @@
...
+        $this->id = $this->createMock(TransitionId::class);
+        $this->from = $this->createMock(StatusCode::class);
+        $this->to = $this->createMock(StatusCode::class);
+        $this->roleIds = [$this->createMock(RoleId::class), $this->createMock(RoleId::class)];
+        $this->conditionSetId = $this->createMock(ConditionSetId::class);
+    }
+
+    /**
+     */
+    public function testMultimediaCreation(): void
```
Comment (rprzedzik) [inline module/event-sourcing/tests/Domain/Entity/TransitionTest.php]: TransitionCreation

File: module/event-sourcing/tests/Domain/Entity/TransitionTest.php [hunk 14]
```diff
@@ -0,0 +1,118 @@
...
+    }
+
+    /**
+     */
+    public function testChangingConditionSetNull(): void
+    {
+        $transition = new Transition($this->id, $this->from, $this->to, $this->roleIds);
+        $this->assertEmpty($transition->changeConditionSetId());
+    }
+
```
Comment (rprzedzik) [inline module/event-sourcing/tests/Domain/Entity/TransitionTest.php]: Unnecessary br

File: module/event-sourcing/tests/Infrastructure/Handler/ChangeProductAttributeValueCommandHandlerTest.php [hunk 15]
```diff
@@ -0,0 +1,137 @@
...
+use Ergonode\Editor\Domain\Entity\ProductDraftId;
+use Ergonode\Editor\Domain\Repository\ProductDraftRepositoryInterface;
+use Ergonode\Editor\Infrastructure\Handler\ChangeProductAttributeValueCommandHandler;
+use Ergonode\Value\Domain\Service\ValueManipulationService;
+use Ergonode\Value\Domain\ValueObject\StringValue;
+use PHPUnit\Framework\TestCase;
+
+/**
+ */
+class ChangeProductAttributeValueCommandHandlerTest extends TestCase
```
Comment (rprzedzik) [inline module/event-sourcing/tests/Infrastructure/Handler/ChangeProductAttributeValueCommandHandlerTest.php]: This class going to be removed with draft, unnecessary test

File: module/event-sourcing/tests/Infrastructure/Handler/ChangeProductAttributeValueCommandHandlerTest.php (hunk 7 above)
Comment (rprzedzik) [inline module/event-sourcing/tests/Infrastructure/Handler/ChangeProductAttributeValueCommandHandlerTest.php]: What is Attr ???

File: module/transformer/tests/Domain/Entity/ProcessorTest.php [hunk 16]
```diff
@@ -0,0 +1,96 @@
...
+    }
+
+    /**
+     * @expectedException \LogicException
+     */
+    public function testProcess(): void
+    {
+        $processor = new Processor($this->id, $this->transformerId, $this->importId, $this->action);
+        $processor->process();
+        $processor->process();
```
Comment (WojFa) [inline module/transformer/tests/Domain/Entity/ProcessorTest.php]: Why you use double `$processor->process();`

File: module/transformer/tests/Domain/Entity/ProcessorTest.php [hunk 17]
```diff
@@ -0,0 +1,96 @@
...
+
+    /**
+     * @expectedException \LogicException
+     */
+    public function testEnd(): void
+    {
+        $processor = new Processor($this->id, $this->transformerId, $this->importId, $this->action);
+        $processor->process();
+        $processor->end();
+        $processor->end();
```
Comment (WojFa) [inline module/transformer/tests/Domain/Entity/ProcessorTest.php]: Why you use double `$processor->end();`

File: module/transformer/tests/Domain/Entity/ProcessorTest.php [hunk 18]
```diff
@@ -0,0 +1,96 @@
...
+    }
+
+    /**
+     * @expectedException \LogicException
+     */
+    public function testStop(): void
+    {
+        $processor = new Processor($this->id, $this->transformerId, $this->importId, $this->action);
+        $processor->stop();
+        $processor->stop();
```
Comment (WojFa) [inline module/transformer/tests/Domain/Entity/ProcessorTest.php]: Why you use double `$processor->stop();`

File: module/transformer/tests/Domain/Entity/ProcessorTest.php (hunk 16 above)
Comment (WojFa) [inline module/transformer/tests/Domain/Entity/ProcessorTest.php]: Why you use double `$processor->process();`

**Pull request merged**